app/views/api/v2/smart_proxies/import_subnets.json.rabl

```diff
@@ -0,0 +1,3 @@
+node :subnets do
+  locals[:result]
+end
```

Comment:
Shouldn't it be:

```ruby
node :results do
  locals[:subnets]
end
```
or even
```ruby
node :results do
  collection @subnets
end

extends "api/v2/subnets/main"
```
?

Reply:
"Rather than result, should be self describing and be named subnets?" 
on previous comment @ekohl commented this, if I understood him correctly he thinks that is a better option. 
I am not sure what is preferred

Reply:
I think passing in `@subnets` is easier to follow. Option 2 that @ofedoren suggested looks cleanest to me. I'll defer to @ofedoren for this since he's much more experienced with the Rails side of an API. My knowledge is much more applicable to REST API design.